ibis/backends/flink/__init__.py

```diff
@@ -331,13 +335,28 @@ def create_table(
             # INTO ... VALUES to keep things consistent
             self._table_env.create_temporary_view(qualified_name, table)
         if isinstance(obj, ir.Table):
-            # TODO(chloeh13q): implement CREATE TABLE for expressions
-            raise NotImplementedError
-        if schema is not None:
-            if not tbl_properties:
-                raise exc.IbisError(
-                    "tbl_properties is required when creating table with schema"
+            if schema is not None and not schema.equals(obj.schema()):
```

Comment:
I think the expected behavior is that "only one of `schema` or `obj` can be provided" (see the docstring)

Reply:
Thanks, but there are several places in the `common tests` where both `schema` and `obj` are provided. We need to decide whether to 
(1) check if only one of them is provided and raise an error otherwise, 
(2) support providing both and check if the schemas match. 
Which one makes more sense to you? (The second increases test coverage.)

Reply:
I checked the implementation in some of the other backends. Did you get this from the BigQuery backend? I guess it's fine if this is how another backend does it

Reply:
Thanks, implemented `option 1`.